students/CarolF/module02/series.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python3
+
+"""
+Week2 Fibonacci Excercise
+Created by : Carol Farris
+Goal: create a function labeled Fibonacci that returns teh nth value in the fibonacci series.
+"""
+
+
+def fibonacci (n):
+    """function creates fibonacci values and prints the nth value in series"""	
+    count = 0 
+    previousNumb = 0
+    currentNumb = 1
+    if  n < 0:
+        return None
+    elif  n == 0:
+        return 0
+    elif  n ==1:
+        return 1
+    else:
+        for i in range (0,n+1):  
+            if i == 0:
+                count = 0
+            if i ==1:
+                count = 1
+                currentNumb = 1
+                previousNumb = 0
+            if i >1:
+                count =  previousNumb + currentNumb
+                previousNumb= currentNumb
+                currentNumb = count
+
+    return(count)
+  
```

Comment:
You can also do something like: 

    return fibonacci(n - 1) + fibonacci(n - 2)